### ISSUE TITLE
CDN call to JQuery - necessary?

### ISSUE THREAD

**Issue by Gavin-McLeod** [bcgov/Gov-2.0-Bootstrap-Skeleton#9]
I see that although the bootstrap library is made local to the project (/js/bootstrap.js and /js/bootstrap.min.js), the same is not true of the JQuery library.  The sample HTML are calling the JQuery library from a CDN (https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js).  

It's easy enough to call JQuery from a local location per project or per organization, but I wonder if there is a particular reason for this?  This won't be acceptable for production code.

**Comment by juhewitt** [comment]
Yup, I'm down with including the core jQuery library rather than linking externally.   Can you do a pull request?
